webdriver/tests/perform_actions/pointer_tripleclick.py

```diff
@@ -0,0 +1,31 @@
+from tests.perform_actions.support.refine import filter_dict, get_events
+from tests.support.asserts import assert_move_to_coordinates
+from tests.support.inline import inline
+
+lots_of_text = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor "\
+               "incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud "\
+               " exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+
+
+def test_tripleclick_at_coordinates(session, mouse_chain):
+    session.url = inline("""<div>
+          {}
+        </div>""".format(lots_of_text))
+    div = session.find.css("div", all=False)
+    div_rect = div.rect
+    div_centre = {
+        "x": div_rect["x"] + div_rect["width"]/2,
+        "y": div_rect["y"] + div_rect["height"]/2
+    }
+    mouse_chain \
+        .pointer_move(div_centre["x"], div_centre["y"]) \
+        .click() \
+        .pause(0) \
```

Comment:
What are the pauses for here?

Reply:
I copied the double click test. I can't see any reason why they should be there for this test so have removed them